runtime/v1/linux/proc/init.go

```diff
@@ -69,7 +69,7 @@ type Init struct {
 	pausing      *atomicBool
 	status       int
 	exited       time.Time
-	pid          safePid
+	pid          int
```

Comment:
I think we can keep `safePid` to  prevent pid's data race and just remove `StoppedPID` and update kill logic. WDYT?

Reply:
This is a backport, so if we want to change that, it's better to make those changes on master, then backport to release branches

Reply:
ok